db/helpers/common.js

```diff
@@ -32,6 +32,54 @@ const changeType = {
 };
 
 /**
+ * Replicated here from api/v1/helpers/verbs/utils.js
+ * Return the sequelize query { name: { $in: arr } }
+ * @param {Array} arr The array to return in the query.
+ * @returns {Object} the query object
+ */
+function whereClauseForNameInArr(arr) {
```

Comment:
if we will be calling the generator models class method from the controller, we might not have to replicate this. We can use the one in the api layer and pass it as another parameter to the custom create method. Corret?

Reply:
I tried requiring the file in the api layer, but there was some error on runtime. Also wouldn't it introduce unnecessary dependencies if we require a file from the api when we are in db? This function is very small.